webpack.common.js

```diff
@@ -9,9 +9,8 @@ module.exports = {
       app: path.join(APP, "src", "js", "app.js"),
    },
    output: {
-      // path: path.join(APP, "..", "web", "assets", "app"),
       path: path.join(APP, "..", "web", "assets", "mobile"),
-      filename: "mobile_[name].js",
+      filename: "mobile_[name].[contenthash].js",
```

Comment:
ok, so here is the thing.  there is an `api_sails/views/mobile_pwa.js` that needs to reference these `.js` files.  If we put `[contenthash]` on here, then that file needs to be updated each time we compile to reference these new `[contenthash]`es.   

But if we leave it off, we can just continue to use `mobile_runtime`, `mobile_app`, `mobile_vendors`  without worrying about updating that file every time.

Reply:
https://github.com/digi-serve/test_pwa/pull/8/commits/80639fb046aee8c1dd84e82e7a92c9d03508eb7b